app/templates/main/download.html

```diff
@@ -1,11 +1,11 @@
 {% extends "base.html" %}
 
 {%- from "govuk_frontend_jinja/components/accordion/macro.html" import govukAccordion -%}
+{%- from 'govuk_frontend_jinja/components/back-link/macro.html' import govukBackLink -%}
```

Comment:
Looks like this redundant line is still here

Reply:
odd - should be good now